src/daemon/ca/certauth.rs

```diff
@@ -798,30 +860,19 @@ impl CertAuth {
             return Err(Error::KeyUseNoIssuedCert);
         }
 
```

Comment:
Is the same functionality retained after this change, i.e. it looks like it was republishing certs on key revocation but no longer does? Is that intentional? Or is this in some way about recording less events but no changing actual functionality?